mysql/datadog_checks/mysql/queries.py

```diff
@@ -74,6 +74,87 @@
 SELECT plugin_status
 FROM information_schema.plugins WHERE plugin_name='group_replication'"""
 
+SQL_DATABASES = """
+SELECT schema_name as `name`,default_character_set_name,default_collation_name FROM information_schema.SCHEMATA
+WHERE schema_name not in ('mysql', 'performance_schema', 'information_schema')"""
+
+SQL_TABLES = """\
+SELECT table_name as `name` FROM information_schema.TABLES WHERE TABLE_SCHEMA = %s
+"""
+
+SQL_COLUMNS = """\
```

Comment:
1. I think several people suggested me that it would be nicer to homogenise across postgres, sqlserver etc.. The most vocal proponent was Joel. That's the main reason. 

2. Yeah, now when I am thinking of it I should've taken a more systematic approach. I discussed certain calls on some one on ones but, probably, the better way would been to provide a link to each table so that each reviewer can check. Let me do that.

Reply:
@justiniso COLUMN_TYPE - same as data_type but with presicion, what if I swap it with data_type in the query and alias it as data_type to make it conform with postgres and sqlserver ? Or more confusing for mysql users that way?

Reply:
Ah, ok, I didn't realize you were trying to homogenize across DBMSs. If they are consistent, then that's okay. A comment might help to explain that and to protect against future contributors changing since aliases in a query are generally just a convenience or for name conflicts.

`COLUMN_TYPE` - precision and length are pretty high value and important for detecting schema drift, i.e. if one database has type `VARCHAR(128)` and another has type `VARCHAR(64)`, then that's a problem for the user because their databases will have different behaviors.

Reply:
> There are indeed query optimizer anomalies where the field length has a huge impact on performance, e.g. https://nenadnoveljic.com/blog/suboptimal-sort-calculation/

Reply:
Yes I think that makes sense, but with the disclaimer that I haven't dug into the values of `data_type` for SQL Server and Postgres. It seems like a reasonable approach though.

Reply:
i suggest to NOT rename `column_type` to `data_type` because `data_type` is also a valid column in the `information_schema.columns` table. Instead, we collect get both `data_type` and `column_type` OR only collect `column_type` not do not rename it. The values of `data_type` columns are inline with other DBMS (postgres, sqlserver) such as integers, text and etc. `column_type` includes the name of the type as well as other information such as length, which is a superset of `data_type`. IMO it is a bit confusing if a column alias to another existing column.

Reply:
ok, I will keep `column_type` as it is then. I think it is better indeed. Yeah, at the end probably we should get back to this idea of unification across sqlserve, postgres , mysql etc... Cause may be for us it's easier to organise things and may be code front end but for the end users it's just gonna be super strange to not to be able to find names that they are used to.